recipes/mvapich2/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "3.0" %}
+
+package:
+  name: mvapich
+  version: {{ version }}
+
+source:
+  url: https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich-{{ version }}.tar.gz
+  sha256: ee076c4e672d18d6bf8dd2250e4a91fa96aac1db2c788e4572b5513d86936efb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  ignore_run_exports:
+    - libhwloc
+    - libnl
+    - libnuma
+    - libzlib
+    - rdma-core
```

Comment:
`ignore_run_exports` is for special circumstances where you need a package in `host` dependencies, but _do not_ link to them. That's pretty rare (e.g. openmpi's optional dependencies), and I don't think it applies here

```suggestion
```

Reply:
OK